ci/terraform/shared/dynamodb.tf

```diff
@@ -111,6 +111,11 @@ resource "aws_dynamodb_table" "user_profile_table" {
     type = "N"
   }
 
+  attribute {
```

Comment:
is this so it can be included in an index?

Reply:
Indeed. It was necessary to declare the attribute here in order to use in the index